QUARKUS-5654.md

```diff
@@ -0,0 +1,34 @@
+# QUARKUS-5654 Update MP REST Client to 4.0
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5654
+
+Upstream PRs:
+- https://github.com/quarkusio/quarkus/pull/43959
+
+## Scope of the testing
+- New overloaded `baseUri(String uri)` method so users don’t have to call URI.create()
+- New `header(String name, Object value)` method for adding headers to Client instances
+- Clarify specification CDI support (`@RestClient` qualifier is not optional)
+- Add multi-part processing example to specification.
+- Clarify the specification that when a client injected as a CDI bean falls out of scope, the client is closed
```

Comment:
These bulltipoints are basically highlights (c&p) from MP Rest Client 4.0 RN, but I wonder how can you test "clarify" without seeing anything documented in linked upstream PRs? I can't find where is clarified that the rest client is closed when bean is destroyed (which is how I understand that statement, e.g. by default app scoped so it is closed when arc container is closed; basically _autoclose, we take care of it_. Where is it https://quarkus.io/guides/rest-client ?

Reply:
Quarkus updated MP Rest client version to 4.0. These 5 bullet points (including "clarifications" and an example ) are changes in MP Rest client itself (and in its specification), and not in Quarkus.